src/Hmvc/src/Interceptors/Handler/ReflectionHandler.php

```diff
@@ -30,7 +31,7 @@ public function __construct(
      * @psalm-assert non-empty-string $action
      * @throws \Throwable
      */
-    public function handle(CallContext $context): mixed
+    public function handle(CallContextInterface $context): mixed
```

Comment:
Perhaps it is also necessary to replace CallContext with CallContextInterface here (line 78)?
```php
protected function resolveArguments(\ReflectionFunctionAbstract $method, CallContext $context): array
```

Reply:
Thanks. Fixed.